EIPS/eip-2309.md

```diff
@@ -46,6 +45,10 @@ The address executing the transaction **MUST** own all the tokens within the ran
 
 The `fromTokenId` and `toTokenId` **MUST** be a consecutive range of tokens IDs.
 
+The `fromTokenId`, `fromAddress`, and `toAddress` **MUST** be indexed parameters
```

Comment:
Nitpicking:
```suggestion
The `fromTokenId`, `fromAddress`, and `toAddress` **MUST** be indexed parameters.
```

Reply:
@pizzarob realised that half of the sentences were terminated with a period, the other half weren't. If you still want to fix that feel free to open a PR, it can be merged as it is only a stylistic change.